docs/docs/labelling-issues.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Labelling issues
+---
+
+## What are issue labels?
+
+Issue labels are a tool in GitHub that that is used to group issues into one or more categories.
+
+[Check out Gatsby's labels (and their descriptions)](https://github.com/gatsbyjs/gatsby/issues/labels)
+
+
+## Why label issues?
+
+Gatsby is a very active project with many new issues opened each day. Labelling issues helps by identifying:
+
+- good issues for new contributors to work on
+- reported and confirmed bugs
+- new features
+- duplicate issues
+- issues that are stalled or blocked
+
+## Who can label issues?
+
+Anyone who's a member of the [Gatsby Maintainers team](https://github.com/orgs/gatsbyjs/teams/maintainers) can label issues.
+
+You can get an invite to the team by having a Pull Request merged into the Gatsby project. Check out the list of [`help wanted`](https://github.com/gatsbyjs/gatsby/labels/%F0%9F%93%8D%20status%3A%20help%20wanted) issues and the [How To Contribute Guide](/docs/how-to-contribute/) to get started.
+
+## How to label an issue
+
+Ideally every issue should have a single `🏷 type:` label applied to it. Optionally a `status: ` label, or other labels may also be applied.
+
+Before continuing, get familiar with [Gatsby's issue labels and their descriptions](https://github.com/gatsbyjs/gatsby/issues/labels).
+
+The broad steps to labelling an issue are:
+
+- Read an issue
+- Choose the labels that apply to that issue
+- That's it
```

Comment:
Can we add an exclamation point or emoji here to make it a bit more playful?

Reply:
I very nearly went for the classic 💜 💪, but in the [style guide](https://next.gatsbyjs.org/docs/gatsby-style-guide/) it says:

> Don’t use emojis or emoticons in the Guide. Gatsby has a global community, and the cultural meaning of an emoji or emoticon may be different around the world. Also, emojis can render differently on different systems.

Maybe we can break that rule occasionally?

Reply:
I attempted to lighten the mood with words instead of emoji here

Reply:
Hm, I have no idea...just looked up this article and wondering how much internet culture reduces some misunderstandings (e.g. if someone grew up using emojis w/ people all across the world, there's a chance all will have a shared understanding. However, it's hard to be sure!) https://www.airportparkingreservations.com/blog/avoid-using-these-hand-gestures-and-emojis-in-other-countries/

^^^ Also I'd have to do some more thorough linguistics research bc that article could be wrong.

Looks like 💪💜 these don't mean anything though!